jbmc/src/java_bytecode/java_bytecode_convert_method.cpp

```diff
@@ -2197,7 +2212,7 @@ void java_bytecode_convert_methodt::convert_invoke(
   //   generate code that may wrongly assume that such a method is
   //   accessible if we assume that its access attribute is "more
   //   accessible" than it actually is.
-  irep_idt id = arg0.get(ID_identifier);
+  const irep_idt id = arg0.get(ID_identifier);
```

Comment:
`const &`?

Reply:
it's not clear for me what `arg0` represents, if you have an intuition about that can you rename the parameter?